src/lib.rs

```diff
@@ -116,10 +114,58 @@ impl Default for EguiSettings {
     }
 }
 
+/// Wrap the `WindowId` to `EguiRenderOutput` hashmap
+#[derive(Resource)]
+pub struct EguiRenderOutputContainer(pub HashMap<WindowId, EguiRenderOutput>);
+
+impl Deref for EguiRenderOutputContainer {
+    type Target = HashMap<WindowId, EguiRenderOutput>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
```

Comment:
You can actually just use Bevy's `Deref` and `DerefMut` derives to achieve this.